## /bin/sh -e simulate dpatch for prepend_chlog.py
## Config
## conf_sample.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
## DP: Neither load chan_oss nor chan_h323 by default in sample conf

diff -urNad asterisk-1.2.7.1.dfsg~/configs/modules.conf.sample asterisk-1.2.7.1.dfsg/configs/modules.conf.sample
--- asterisk-1.2.7.1.dfsg~/configs/modules.conf.sample	2005-11-29 18:24:39.000000000 +0000
+++ asterisk-1.2.7.1.dfsg/configs/modules.conf.sample	2006-04-21 17:28:48.000000000 +0100
@@ -45,7 +45,10 @@
 ; By default, load OSS only (automatically) and do not load ALSA
 ;
 noload => chan_alsa.so
-;noload => chan_oss.so
+noload => chan_oss.so
+;
+; h323 modules can be unstable so don't load by default
+noload => chan_h323.so
 ;
 ; Module names listed in "global" section will have symbols globally
 ; exported to modules loaded after them.
